### DISCUSSION TITLE
blog/why-roam-research

### DISCUSSION THREAD

**Discussion by giscus** [Pathsis/2750-words#10] [Announcements]
# blog/why-roam-research

使用双链笔记的缘由和隐忧。

https://rercel.com/blog/why-roam-research

**Comment by sifeano** [comment]
你好，在笔记方面，想请教你一些问题，也是困扰我很久的问题：
1. 如何将笔记运用到论文的写作中？我好像记了不少笔记，但是没法转化成文章。
2. 有时候读一篇文献可能会横跨一段时间，或者是隔了一段时间再读，如果用daily notes来写笔记，那么这时候是在同一个文档写一篇文献的笔记，还是分别在每天的daily notes上写？
3. 是否方便介绍你的工作流？
谢谢，叨扰了！

**Reply by Pathsis** [reply]
你好，我最近写了一篇文章，介绍了[使用 Roam Research 的实例](https://pathos.page/blog/how-to-use-roam-research)，共你参考。

**Comment by chaoliu719** [comment]
博主还在用 roam 吗？价格觉得可以接受吗？logseq 从 github 上来看基本都是一个老外在维护，一些小问题也没空修复。所以来问问现在还推荐 roam 吗

**Reply by Pathsis** [reply]
我目前还在用Roam。Roam是在太好用了。对于Logseq，我觉得它很有可能成为一个烂尾楼，无穷无尽的bug，细节特别差，我曾在他们的反馈渠道多次抱怨他们没有一个合理的开发计划和软件质量保障。不够话说回来了，Logseq可用。

在Logseq可用的情况下，在选择问题上，值得考虑的最重要问题之一是软件的**持久性**。Logseq本地化，无需登录使用，这是相比Roam最重要的优势。假如Roam停止了开发，或者停止了服务器，无法登录Roam就无法继续再使用。即使将其数据导出，再转成其他软件的数据，肯定支离破碎。

另外，你完全可以免费使用Roam（本地库），看起来官方是默许的，但不做任何保障而已。
